tools/cooja/build.gradle

```diff
@@ -0,0 +1,88 @@
+allprojects {
+  apply plugin: 'java'
+
+    repositories {
+      jcenter()
+    }
+
+}
+
+apply plugin: 'findbugs'
+apply plugin: 'checkstyle'
+apply plugin: 'application'
+apply plugin: 'eclipse'
+apply plugin: 'idea'
```

Comment:
Ah sorry, that was badly phrased. I was wondering whether only Cooja is checker using checkstyle abd findbugs, or  the Cooja-Plugins, too.